Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/Physics/SphereCollider.cpp

```diff
@@ -0,0 +1,15 @@
+#include "SphereCollider.hpp"
+
+namespace saintNecEngine
+{
+
+    namespace saintNecPhysics
+
+    {
+        SphereCollider::SphereCollider(Vector3 center, double radius)
```

Comment:
For this one i prefer to keep it liek that for the following reason:

If u call this constructor like SphereCollider(Vector3{....}, ...) ; as the point is created in the call and it is not a const parameters, it will make an error "lvalue...". However, using like it will call the copy constructor of Vector 3 and fill the variable

Reply:
(Not sure it does make the same with initialisation list; I must check it)